commands/run.go

```diff
@@ -46,7 +46,7 @@ func Run(argv []string) {
 
 	// Parse args for command provider
 	subArguments, _ := docopt.Parse(
-		provider.Usage(), arguments["<args>"].([]string),
+		provider.Usage(), append([]string{cmd}, arguments["<args>"].([]string)...),
```

Comment:
I can get help working... I have testing the `work` command...
Maybe I need to figure out how it works...